.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015"
```

Comment:
can we also add "react" here? We don't have any jsx files yet, so we can add it then.

Reply:
I think i want to wait, because this babelrc is actually affecting our babel translator for the mocha. So basically this is running on like capture-suite.js

Reply:
Ya, but I don't think it affects .js files that don't have JSX in it. It basically compiles the JSX -> JS. Although I'm testing the branch again right now and having issues.